### DISCUSSION TITLE
Warning MQTT naming with update to Home Assistant 2023.8.1

### DISCUSSION THREAD

**Discussion by pphil78** [mak-gitdev/HA_enoceanmqtt#76] [General]
Hello @mak-gitdev
With the last update to Home Assistant 2023.8.1 warning appears : 

"Entités MQTT découvertes avec un nom commençant par le nom de l'appareil"
Ceci ne fonctionnera plus dans la version 2024.2.0. Veuillez y remédier
"Certaines entités MQTT ont un nom d'entité qui commence par le nom de l'appareil. Cela ne devrait pas être le cas. Pour éviter un nom en double, le préfixe du nom de l'appareil est supprimé du nom de l'entité en guise de solution de contournement. Veuillez informer le mainteneur de l'application logicielle qui fournit les entités concernées pour résoudre ce problème."

and

"Entités MQTT découvertes avec un nom identique au nom de l'appareil"
Ceci ne fonctionnera plus dans la version 2024.2.0. Veuillez y remédier
"Certaines entités MQTT ont un nom d'entité identique au nom de l'appareil. Cela ne devrait pas être le cas. Le nom de l'entité est défini sur "null" comme solution de contournement pour éviter un nom en double. Veuillez informer le mainteneur de l'application logicielle qui fournit les entités concernées pour résoudre ce problème"

This is related to https://developers.home-assistant.io/blog/2023-057-21-change-naming-mqtt-entities/

Regards
Philippe

**Comment by mak-gitdev** [comment]
Hi @pphil78,

I am aware of this warning.
The next release will "fix" it.
I think I will now check the version of HA to know how to generate entity name.
And add a new configuration entry to let the users select what to do for versions from 2023.8.1 to 2024.2.0.

**Comment by mak-gitdev** [comment]
[Done](https://github.com/users/mak-gitdev/projects/1/views/1?pane=issue&itemId=35566401) in [0.1.27](https://github.com/mak-gitdev/HA_enoceanmqtt/commit/0d0a4cdb3d74b191feb0cc9169e11f10243d1ec9)
